bin/danger.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+[ ! -z $DANGER_GITHUB_API_TOKEN ] && bundle exec danger --verbose || echo "Skipping Danger for External Contributor"
```

Comment:
Would this print "Skipping Danger For External Contributor" if `bundle exec danger --verbose` fails for any reason? 🤔 My bash scripting skills are very weak I admit.

Reply:
They don't say much about it on their website, but I _suspect_ that it means "If you don't have the token `and` execute this command, print that"...

Reply:
I guess I'm more wondering about the case where the token is present, but `bundle exec danger` returns a `false` value for some reason.

Reply:
Good catch @ifbarrera, I've updated the script to fail under normal circumstances with:

<img width="605" alt="image" src="https://user-images.githubusercontent.com/3735375/69451709-e292cd00-0d14-11ea-95ca-d10cf5522725.png">

Otherwise, yes - it would just say "Skipping Danger For External Contributor" for any failure reason.